app/middlewares/debuggerAPI.js

```diff
@@ -43,14 +55,22 @@ const workerOnMessage = message => {
   socket.send(JSON.stringify(data));
 };
 
+const onWindowMessage = e => {
+  const {data} = e;
```

Comment:
if we receive data from panel in web part, we forward the message to the worker. There are some checks for payload, but they are there since apollo dev tools don't strictly follow api for event emitter.